contracts/tasks/runner/prove.ts

```diff
@@ -145,7 +135,16 @@ task("prove", "Command to generate proof and prove the result of a poll on-chain
         key: `poll-${poll.toString()}`,
       });
       const tallyContractAddress = await tallyContract.getAddress();
-
+      const useQuadraticVoting =
+        deployment.getDeployConfigField<boolean | null>(EContracts.Poll, "useQuadraticVoting") ?? false;
+      const mode = useQuadraticVoting ? "qv" : "nonQv";
+      const tallyZkey = deployment.getDeployConfigField<string>(EContracts.VkRegistry, `zkeys.${mode}.tallyVotesZkey`);
+      const tallyWasm = deployment.getDeployConfigField<string>(EContracts.VkRegistry, `zkeys.${mode}.tallyWasm`);
+      const processZkey = deployment.getDeployConfigField<string>(
+        EContracts.VkRegistry,
+        `zkeys.${mode}.processMessagesZkey`,
+      );
+      const processWasm = deployment.getDeployConfigField<string>(EContracts.VkRegistry, `zkeys.${mode}.processWasm`);
```

Comment:
For these fields you need to pass mustGet = true for `getDeployConfigField` method.

Reply:
fixed! thanks!